applications/ShapeOptimizationApplication/tests/sensitivity_verification_process_test/run_semi_analytic_step_size_verification.py

```diff
@@ -93,4 +93,4 @@
     kratos_utilities.DeleteDirectoryIfExisting("__pycache__")
     kratos_utilities.DeleteFileIfExisting("sensitivity_verification_process_test.post.lst")
     kratos_utilities.DeleteFileIfExisting(results_filename)
-    kratos_utilities.DeleteFileIfExisting("structure.post.bin")
\ No newline at end of file
+    kratos_utilities.DeleteFileIfExisting("structure.post.bin")
```

Comment:
idem

Reply:
I see the point, however this would be something for a new MR, as this is just a whitespace change here and not related.